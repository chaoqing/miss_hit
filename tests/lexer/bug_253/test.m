[-.1 -~0] % should be [-.1 -1]
