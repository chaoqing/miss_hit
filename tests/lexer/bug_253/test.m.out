In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
| ^ info: [SL] M_BRA
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|  ^ info: OPERATOR
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|   ^^ info: NUMBER
In test.m, line 1
| [-.1<anon,> -~0] % should be [-.1 -1]
|     ^^^^^^^ info: COMMA
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|      ^ info: OPERATOR
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|       ^ info: OPERATOR
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|        ^ info: NUMBER
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|         ^ info: M_KET
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|           ^^^^^^^^^^^^^^^^^^^^ info: COMMENT 'should be [-.1 -1]'
In test.m, line 1
| [-.1 -~0] % should be [-.1 -1]
|                               ^ info: NEWLINE
test.m: lexed OK
MISS_HIT Debug Summary: 1 file(s) analysed, everything seems fine
